CLIENT-SPECIFICATION.md

```diff
@@ -116,7 +116,12 @@ The structure inside these translation folders is identical to that of the main
 
 ## Page structure
 
-Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), which the exception of the non-standard `{{` and `}}` syntax, which surrounds values in an example that users may edit. Clients MUST NOT break if the page format is changed within the _CommonMark_ specification.
+Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), with the exception of the non-standard `{{` and `}}` placeholder syntax, which surrounds values in an example that users may edit. Clients MAY highlight the placeholders and MUST remove the surrounding curly braces. Clients MUST display the `{{` and `}}` if escaped using `\` (i.e. `\{\{` and `\}\}`). Clients MUST NOT break if the page format is changed within the _CommonMark_ specification.
```

Comment:
> Clients MUST display the `{{` and `}}` if escaped using `\` (i.e. `\{\{` and `\}\}`).

I think it'd be better to say it's not considered the placeholder syntax if the curly braces are escaped.

Only saying the curly braces must be displayed, does not mean it's not a placeholder. It just sounds like it's a placeholder that should have it's curly braces displayed for some reason. I think the intent was to say it's not a placeholder at all.

Maybe something like:

```suggestion
Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), with the exception of the non-standard `{{` and `}}` placeholder syntax, which surrounds values in an example that users may edit. Clients MAY highlight the placeholders and MUST remove the surrounding curly braces. Clients MUST not treat `{{` and `}}` as the placeholder syntax if it is escaped using `\` (i.e. `\{\{` and `\}\}`). Clients MUST NOT break if the page format is changed within the _CommonMark_ specification.
```

>

Reply:
What about this?

```suggestion
Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), with the exception of the non-standard `{{` and `}}` placeholder syntax, which surrounds values in an example that users may edit. Clients MAY highlight the placeholders and MUST remove the surrounding curly braces. Clients MUST NOT treat them as the placeholder syntax if they are escaped using `\` (i.e. `\{\{` and `\}\}`) and MUST instead display literal braces, without backslashes. Clients MUST NOT break if the page format is changed within the _CommonMark_ specification.
```

Reply:
I think your version is great, nice one.